test/cli/samples/config-promise/_expected.js

```diff
@@ -0,0 +1,11 @@
+(function (global, factory) {
```

Comment:
Including this file does not have an effect as it is only checked if you do NOT set `execute: true` in `_config.js`. You should remove it as it is rather confusing (and while you are at it, remove it from the `cli/config` test as well 😜)

Reply:
You got it!